app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  has_many :posts, foreign_key: 'author_id'
```

Comment:
- [ ] Good job setting the foreign key for posts as the previous reviewer requested, however, you would also need to set it for likes and comments too. e.g:
``` 
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
has_many :posts, foreign_key: 'author_id'